src/test/java/com/targetReady/BookMaster/BookMasterApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package com.targetReady.BookMaster;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class BookMasterApplicationTests {
+
+	@Test
+	void contextLoads() {
```

Comment:
Please auto format all the files IntelliJ -> Code -> Reformat Code.

Reply:
done with all the changes